### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by CardWasDeclined** [frazanolo/discord-success-poster-bot#1]
NameError: name 'tweet_id' is not defined

**Comment by frazanolo** [comment]
> NameError: name 'tweet_id' is not defined

My bad, check now it should be ok!

**Comment by CardWasDeclined** [comment]
Yes it's worked but it only delete the tweet won't delete picture on discord

**Comment by frazanolo** [comment]
> Yes it's worked but it only delete the tweet won't delete picture on discord

Yeah that's not important as the user that posted it can easily delete it whenever he wants (on discord)

**Comment by CardWasDeclined** [comment]
Ok in line 13 it should be load not loads
